crates/sui-adapter-transactional-tests/tests/internal/internal_ptb_fail.move

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// Case: try to call internal in PTB, fail
+
+//# init --addresses test=0x0 --accounts A
+
+//# publish
+module test::custom_type {
+    public struct CustomType {}
+}
+
+//# programmable --sender A
+//> 0: std::internal::permit<test::custom_type::CustomType>();
```

Comment:
Can you also add a couple tests (both here and for calls in Move) that try to call this with a native type (e.g, u64, etc)

Reply:
Yup, added!